src/mpi/init/mpir_init.c

```diff
@@ -79,12 +79,18 @@ int MPIR_Init_impl(int *argc, char ***argv)
     }
 
     int provided;
-    mpi_errno = MPIR_Init_thread_impl(argc, argv, threadLevel, &provided);
+    mpi_errno = MPII_Init_thread(argc, argv, threadLevel, &provided, NULL);
 
     return mpi_errno;
 }
 
 int MPIR_Init_thread_impl(int *argc, char ***argv, int user_required, int *provided)
+{
+    return MPII_Init_thread(argc, argv, user_required, provided, NULL);
+}
+
+int MPII_Init_thread(int *argc, char ***argv, int user_required, int *provided,
+                     MPIR_Session * session_ptr)
 {
```

Comment:
It looks like the renaming of `MPIR_Init|Finalize` calls have been updated also in a previous commit
```commit 43de3764a9680b2bff46d8c96b709769bc5a2feb
Author: Hui Zhou <hzhou321@anl.gov>
Date:   Tue Dec 1 22:48:02 2020 -0600

    mpi/init: clean up init impl functions
```

Can you merge them?

Reply:
Besides, can you please explain why you define both `MPIR_Init_thread_impl` and `MPII_Init_thread`? It looks like `MPIR_Init_thread_impl` is not used anywhere. Same question to `MPIR_Finalize_impl`.

Reply:
`MPIR_Init_thread_impl` and `MPIR_Finalize_impl` is called by `MPI-` functions (`MPI_Init_thread` and `MPI_Finalize`). The MPI functions are generated by the python binding scripts, thus you couldn't find them before `autogen`.

Reply:
> 
> 
> It looks like the renaming of `MPIR_Init|Finalize` calls have been updated also in a previous commit
> 
> ```
> Author: Hui Zhou <hzhou321@anl.gov>
> Date:   Tue Dec 1 22:48:02 2020 -0600
> 
>     mpi/init: clean up init impl functions
> ```
> 
> Can you merge them?

I think you are referring to a commit in this PR https://github.com/pmodels/mpich/pull/4964 that is already merged.